internal/dkg/store.go

```diff
@@ -24,42 +24,107 @@ type BoltStore struct {
 }
 
 const BoltFileName = "dkg.db"
+const dkgFileName = "dkg.toml"
+const dkgStagedFileName = "dkg.staged.toml"
 const BoltStoreOpenPerm = 0660
 const DirPerm = 0755
 
 var stagedStateBucket = []byte("dkg")
```

Comment:
can probably delete these bucket names

Reply:
if I remove this I have to remove all the boltdb methods. So if you're sure the migration will work from group file for people already using boltdb then sure.....

Reply:
I don't quite follow what you mean

Reply:
NVM. I ripped out all the boltdb code and it builds so clearly not needed at all.